zerver/migrations/0405_set_default_for_enable_read_receipts.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.6 on 2022-08-08 16:52
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_default_for_enable_read_receipts(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    # We enable read receipts by default in realms which require an invitation to join.
+    Realm.objects.filter(invite_required=True).update(enable_read_receipts=True)
```

Comment:
An organization is not open if `emails_restricted_to_domains || invite_required`; so I think we want;
`Realm.objects.exclude(invite_required=False).exclude(emails_restricted_to_domains=False).update(enable_read_receipts=True)`

for this query.